pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -256,7 +256,8 @@ def get_evaluate_epoch_results(self, test_mode):
             print('-' * 80)
             for result_idx, results in enumerate(self.eval_loop_results):
                 print(f'DATALOADER:{result_idx} TEST RESULTS')
-                pprint(results)
+                pprint({k: (v.item() if v.numel() == 1 else v.tolist()) if isinstance(v, torch.Tensor) else v
+                        for k, v in results.items()})
```

Comment:
Just thinking if it is worth to actually do our own pprint, that does this kind of unwrapping for tensors so that we can simply reuse this one (also nice for user)